spec/service_models/miq_ae_service_ems_cluster_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe MiqAeMethodService::MiqAeServiceEmsCluster do
+  let(:cluster) { FactoryGirl.create(:ems_cluster) }
+  let(:svc_cluster) { MiqAeMethodService::MiqAeServiceEmsCluster.find(cluster.id) }
+
+  it "#show_url" do
+    ui_url = "https://www.example.com"
+    miq_region = FactoryGirl.create(:miq_region)
+    allow(MiqRegion).to receive(:my_region).and_return(miq_region)
+    allow(miq_region).to receive(:remote_ui_url).and_return(ui_url)
```

Comment:
This setup is copy-pasted many times and doesn't use anything from the let blocks, so I think it's better to have this as a helper method in some spec/support file